app/core/api/users.coffee

```diff
@@ -8,6 +8,30 @@ module.exports = {
 
   getByEmail: (email, options={}) ->
     fetchJson("/db/user", _.merge {}, options, { data: { email } })
+    
+  getForClassroom: (classroom, options) ->
+    removeDeleted = options.removeDeleted
+    delete options.removeDeleted
+    classroomID = classroom._id or classroom
+    limit = 10
+    skip = 0
+    size = _.size(classroom.members)
+    url = "/db/classroom/#{classroomID}/members"
+    options.data ?= {}
+    options.data.memberLimit = limit
+    options.remove = false
+    jqxhrs = []
+    while skip < size
+      options = _.cloneDeep(options)
+      options.data.memberSkip = skip
+      jqxhrs.push(fetchJson(url, options))
+      skip += limit
+    return Promise.all(jqxhrs).then (data) ->
+      users = _.flatten(data)
+      if removeDeleted
+        users = _.filter users, (user) ->
+          not user.deleted
+      return users
```

Comment:
I think this belongs in the `classrooms` api file, basing it on the root of the URL, rather than the return value. I could go either way, really, but we should be consistent. I lean more toward basing on URL if only because return values could theoretically be a combination of models or data generated from the server but doesn't exist as a collection. Whereas by url, it's very simple to figure out what "group" an endpoint is in.